docs/source/release/v6.13.0/Workbench/New_features/38080.rst

```diff
@@ -0,0 +1 @@
+Mantid's Python code editor now supports F-string syntax highlighting
```

Comment:
```suggestion
- The :ref:`Python script editor <WorkbenchScriptWindow>` now supports f-string syntax highlighting.
```
Add bullet point, add full stop, add link to script editor and remove capital f (https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals)